frontend/metadata/types.go

```diff
@@ -0,0 +1,3 @@
+package frontendmeta
+
+const AttrsKey = "internal.frontend.attrs"
```

Comment:
Why is this in "internal" scope and defined separately? It should be consistent with the way buildinfo sources are handled. Both of them are:
- optionally created in dockerfile frontend
- llb builder wraps and modifies the result
- end up in result metadata
- optionally end up inlined inside image

The LLB wrapping part is different but for the rest as much as we can keep them similar and together the better.